packages/es-components/src/components/containers/sliding-pane/SlidingPaneStyles.js

```diff
@@ -0,0 +1,140 @@
+import { createGlobalStyle } from 'styled-components';
```

Comment:
I'd really like to see this component converted to use styled-components rather than relying on `createGlobalStyle`.

Reply:
I know we want to move away from the global styling, but a lot of this is to hook into `react-modal`'s css. I'm actually not sure how much of this can be ported over to actual styled components.

Reply:
I attempted to move over what I could, but it did not go well. I think we'll have to keep the global styles for now.

Reply:
Ah, I didn't realize react-modal was dependent on these classes. That makes more sense. That will unfortunately also make converting our other modal from `react-overlay` to `react-modal`  very difficult, if that's a direction we wanted to go (I would like to get off that dependency, it's the one that's caused us to use IE polyfills).